editor/src/components/canvas/canvas-strategies/strategies/keyboard-reorder-strategy.ts

```diff
@@ -35,7 +35,7 @@ export function keyboardReorderStrategy(
     return null
   }
   const target = selectedElements[0]
-  const siblings = MetadataUtils.getSiblingsUnordered(canvasState.startingMetadata, target).map(
+  const siblings = MetadataUtils.getSiblingsOrdered(canvasState.startingMetadata, target).map(
```

Comment:
Aha! So this PR actually contained a fix! Please call that out in the PR description and title, since this is actually now the more important part of the PR wrt actual logic changes